src/main/validators/formValidator.ts

```diff
@@ -45,7 +46,27 @@ export class FormValidator {
         if (errors && errors[0] && errors[0].message) {
           const message = `${key}.${errors[0].message}`;
           item._errors.push(message);
-          validationErrors[item.id] = message;
+          if (item._flagComment) {
+            validationErrors[`flagComment-${item.id}`] = message;
+          } else if (
+            item._listOfValuesLength > 0 &&
+            item._listOfValuesLength > JSON.parse(config.get('radio.listOfValuesLength')) &&
+            !item._other
+          ) {
+            validationErrors['custom-accessible-autocomplete'] = message;
+          } else if (
+            item._listOfValuesLength > 0 &&
+            item._listOfValuesLength >= JSON.parse(config.get('radio.listOfValuesLength')) &&
```

Comment:
Is it correct for this condition to be different from the one above `>` vs `>=`?

Reply:
Created this one